dagster_scaleway/serverless_job_launcher.py

```diff
@@ -303,18 +310,18 @@ def check_run_worker_health(self, run: DagsterRun):
         job_run = self._get_scaleway_job_run_from_dagster_run(run)
         if job_run is None:
             return CheckRunHealthResult(
-                WorkerStatus.NOT_FOUND,
+                status=WorkerStatus.NOT_FOUND,
                 run_worker_id=run.run_id,
                 msg=f"Unable to find Scaleway job run with id {run.run_id} for Dagster run {run.run_id}",
             )
 
-        health = CheckRunHealthResult(run_worker_id=run.run_id)
-        health.transient = job_run.state in scw.JOB_RUN_TRANSIENT_STATUSES
-        health.status = SERVERLESS_JOBS_STATES_TO_WORKER_STATUS.get(
-            job_run.state, WorkerStatus.UNKNOWN
+        health = CheckRunHealthResult(
+            status=SERVERLESS_JOBS_STATES_TO_WORKER_STATUS.get(
+                job_run.state, WorkerStatus.UNKNOWN
+            ),
+            run_worker_id=run.run_id,
+            msg=job_run.error_message,
+            transient=job_run.state in scw.JOB_RUN_TRANSIENT_STATUSES,
         )
 
-        if job_run.error_message:
-            health.msg = job_run.error_message
```

Comment:
I don't know I just missed it or there were some changes done to typing in dagster and/or the SDK, but both `msg` and `error_message` are nullable strings `None | str`